docs/cli/call.md

```diff
@@ -5,13 +5,13 @@ sidebar_position: 3
 description: "wash call command reference"
 --- 
 
-Wash call can be used to directly invoke an actor. This can be useful when debugging, especially when the actor isn't directly accessible via external channels such as HTTP. It is not recommended to use wash call in production environments.
+`wash call` can be used to directly invoke a component. This can be useful when debugging, especially when the component isn't directly accessible via external channels such as HTTP. It is not recommended to use `wash call` in production environments.
```

Comment:
We'll need to update this example just cause wash call changed, but this change looks great

Reply:
pushed an update